array-manipulation-2/src/chunk.js

```diff
@@ -0,0 +1,26 @@
+/* exported chunk */
+
+function chunk(array, size) {
+  var mainArr = [];
+
+  for (var i = 0; i < array.length; i++) {
+
+    if (i === 0) {
+      mainArr.push(array.slice(i, i + size));
+    } else if (size === 1) {
+      mainArr.push(array.slice(i, i + size));
+    } else if (i === size) {
+      mainArr.push(array.slice(i, i + size));
+    } else if (i === size * 2) {
+      mainArr.push(array.slice(i, i + size));
+    }
```

Comment:
Every block of this conditional has the same work. You can just remove the conditional and do the same work on every iteration.